bower.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "o-icons": "^4.4.2",
     "o-colors": "^3.3.0",
-    "o-fetch-jsonp": "^2.0.0"
+    "o-fetch-jsonp": "^2.0.0",
+		"o-typography": "^4.3.1"
```

Comment:
I think this should change back to spaces.
`bower.json` is handled by `bower install` which I believe forces spaces much like `npm install` does.